### PR TITLE
rebuild all components

### PR DESCRIPTION
**Pull request by marioferh**

### CODE REVIEW

**Comment by danielmellado** [comment]
Note, this is a temporary trigger to actually allow syft config to propagate to all the components.

**Comment by marioferh** [comment]
Yes, we will revert after the test

**Comment by jan--f** [comment]
wow this is what's needed to rebuild?? Let me try to fix our bundle references first. I want to see if the nudging works as expected.

**Comment by danielmellado** [comment]
@jan--f sure! rebuild button from the UI also doesn't work. LIVING THE DREAM xD

File: Dockerfile.korrel8r [hunk 1]
```diff
@@ -10,7 +10,7 @@ ENV GOEXPERIMENT=strictfipsruntime
 
 RUN go build -tags strictfipsruntime ./cmd/korrel8r
 
-
+//
```
Comment (jan--f) [inline Dockerfile.korrel8r:13]:
```suggestion

```
This causes the build failure

**Review by jan--f — CHANGES_REQUESTED** [review]

**Pull request merged**